.github/workflows/build_and_release.yml

```diff
@@ -470,31 +470,44 @@ jobs:
 
 
       - name: Setup our dev environment
-        uses: ilammy/msvc-dev-cmd@v1.4.1
+        uses: ilammy/msvc-dev-cmd@v1
 
       - name: Setup ninja cmake generator
         uses: seanmiddleditch/gha-setup-ninja@master
 
       - name: Download and build our dependencies using vcpkg
-        uses: lukka/run-vcpkg@v10.2
+        uses: Lord-Kamina/vcpkg-action@master
         with:
-          vcpkgGitCommitId: '14e7bb4ae24616ec54ff6b2f6ef4e8659434ea44'
-          runVcpkgInstall: true
+          pkgs: aubio boost-iostreams boost-locale boost-program-options boost-system cairo cpprestsdk ffmpeg fftw3 fmt fontconfig gdk-pixbuf gettext glibmm glm gtest icu libepoxy librsvg libsigcpp libxml2 libxmlpp nlohmann-json openblas opencv openssl pango portaudio portmidi pthreads sdl2
+          triplet: x64-windows-release
+          extra-args: --clean-after-build
+          cache-key: win64-vcpkg
```

Comment:
Personally i'd just run several bash/powershell commands to download, setup and build vcpkg instead of a github action. This way it's clear for everyone what's happening in the background and probably easier to debug aswell since the action can contain bugs/failures when vcpkg decides to update something

Reply:
Related again to above. By not specifying a git ref, my version will always use the latest stable release. One should hope that stable releases will be... well, stable.